main.go

```diff
@@ -96,6 +96,9 @@ const (
 	cleanOlderThan = 30 * time.Minute
 )
 
+// Pointer to the rules service needed at the time of registration
+var rulesSvc *ia.RulesService
```

Comment:
no reason to use global variable here.

Reply:
It is not global, it is only accessible within the module.
The variable is initialized in main() and used in runGRPCServer(). How would you like to do that?

Reply:
Like we do with other services, please read the code.

Reply:
@yareach pass it form `main` to `runGRPCServer()` with `gRPCServerDeps`. Same as we do for `alertsService` for example.